### PR TITLE
Fix distributions and graphics

### PR DESCRIPTION
**Pull request by mbaudin47**
* Added new example for survival distribution and sample.
* Improved the example to draw an empirical CDF.

### CODE REVIEW

File: python/doc/examples/data_analysis/data_analysis.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Data analysis
 =============
 
+Analyze a sample
```
Comment (jschueller) [inline python/doc/examples/data_analysis/data_analysis.rst]: Sample analysis

Comment (mbaudin47) [reply python/doc/examples/data_analysis/data_analysis.rst]: Ok.

File: python/doc/examples/data_analysis/draw_survival.ipynb [hunk 2]
```diff
@@ -0,0 +1,644 @@
...
+    "for any $x\\in\\mathbb{R}$. \n",
+    "The survival function $S$ is:\n",
+    "$$\n",
+    "S(x) = P(X>x) = 1 - P(X\\leq x) = 1 - F(x)\n",
+    "$$\n",
+    "for any $x\\in\\mathbb{R}$. \n",
+    "\n",
+    "Let's assume that $\\{x_1,...,x_N\\}$ is a sample from $F$. \n",
+    "\n",
+    "Let $\\hat{F}_N$ be the empirical distribution function:\n",
```
Comment (sofianehaddad) [inline python/doc/examples/data_analysis/draw_survival.ipynb]: empirical cumulative distribution function

File: python/doc/examples/data_analysis/draw_survival.ipynb [hunk 3]
```diff
@@ -0,0 +1,644 @@
...
+    "$$\n",
+    "\\hat{F}_N(x) = \\frac{1}{N} \\sum_{i=1}^N \\mathbb{1}_{x_i\\leq x}\n",
```
Comment (regislebrun) [inline python/doc/examples/data_analysis/draw_survival.ipynb]: \mathbb{1} is not well-defined (see [here](https://tex.stackexchange.com/questions/488/blackboard-bold-characters/3260). Either you manage to implement one of the solutions given in the link in the context of a notebook, or you resort to another way to display the indicator function, e.g `\\mathbf{1}`

File: python/doc/examples/data_analysis/draw_survival.ipynb (hunk 2 above)
Comment (mbaudin47) [reply python/doc/examples/data_analysis/draw_survival.ipynb]: Ok: fixed.

File: python/doc/examples/data_analysis/visualize_empirical_cdf.ipynb [hunk 4]
```diff
@@ -40,997 +36,59 @@
...
-       "    <!-- X0 CDF -->\n",
-       "    <defs>\n",
-       "     <path id=\"BitstreamVeraSans-Roman-20\"/>\n",
-       "    </defs>\n",
-       "    <g transform=\"translate(199.305 23.8)scale(0.12 -0.12)\">\n",
-       "     <use xlink:href=\"#BitstreamVeraSans-Roman-58\"/>\n",
-       "     <use x=\"68.505859375\" xlink:href=\"#BitstreamVeraSans-Roman-30\"/>\n",
-       "     <use x=\"132.12890625\" xlink:href=\"#BitstreamVeraSans-Roman-20\"/>\n",
-       "     <use x=\"163.916015625\" xlink:href=\"#BitstreamVeraSans-Roman-43\"/>\n",
-       "     <use
```
[the remark this answers is not in the document]
Comment (mbaudin47) [reply python/doc/examples/data_analysis/visualize_empirical_cdf.ipynb]: That's true: fixed.

File: python/doc/examples/data_analysis/draw_survival.ipynb (hunk 3 above)
Comment (mbaudin47) [reply python/doc/examples/data_analysis/draw_survival.ipynb]: I did'nt know; this is ugly. Let's go for mathbf.

**Pull request merged**